charts/newrelic-infrastructure/values.yaml

```diff
@@ -130,6 +130,16 @@ kubelet:
     scraperMaxReruns: 4
   # port:
   # scheme:
+  # Set enableWindows to true to enable monitoring on Windows nodes
```

Comment:
Could you modify the comments for `enableWindows` and `enableWindows` to use the same formatting as the rest of the file? 
Once you've modified the formatting can you run `helm-docs` tool to automatically update the README.md file?